app/src/main/java/ch/epfl/sdp/peakar/general/InitActivity.java

```diff
@@ -54,8 +60,8 @@ protected void onCreate(Bundle savedInstanceState) {
      * Init application global stuff before opening the main menu
      */
     private synchronized void initApp(){
-        FirebaseAccount firebaseAccount = FirebaseAccount.getAccount();
-        firebaseAccount.synchronizeUserProfile();
+        //FirebaseAccount firebaseAccount = FirebaseAccount.getAccount();
+        //firebaseAccount.synchronizeUserProfile();
```

Comment:
```suggestion
```

Reply:
This was for debugging purpose. I will add them again in next commit